ancestrymanager/ancestrymanager_test.go

```diff
@@ -22,130 +28,787 @@ func newTestResourceManagerClient(opts []option.ClientOption) *cloudresourcemana
 	return rm
 }
 
-func TestAncestryPath(t *testing.T) {
-	cases := []struct {
-		name           string
-		input          []*cloudresourcemanager.Ancestor
-		expectedOutput string
-	}{
-		{
-			name:           "Empty",
-			input:          []*cloudresourcemanager.Ancestor{},
-			expectedOutput: "",
-		},
-		{
-			name: "ProjectOrganization",
-			input: []*cloudresourcemanager.Ancestor{
-				{
-					ResourceId: &cloudresourcemanager.ResourceId{
-						Id:   "my-prj",
-						Type: "project",
-					},
-				},
-				{
-					ResourceId: &cloudresourcemanager.ResourceId{
-						Id:   "my-org",
-						Type: "organization",
-					},
-				},
-			},
-			expectedOutput: "organization/my-org/project/my-prj",
-		},
-	}
+type testTFData struct {
```

Comment:
It might be worth using the [FakeResourceData](https://github.com/GoogleCloudPlatform/terraform-validator/blob/5526b84757265f03cd188fb6129844c0849dfa17/converters/google/fake_resource_data.go#L132) struct here to maximize similarity to behavior from tf data created by the converter - does that seem reasonable?

Reply:
answering my own question: having testTFData here is better. Allows for compact definition without requiring a provided schema. The behavior might be slightly different for edge cases but probably none of those apply for the cases represented by these tests.

Reply:
it causes some circular dependency. 
fake resource data is in converters/google package, and convert.go reference ancestry manager package. If ancestry manager uses fake resource data, then it needs to import fake resource data. 
I move it to another folder/package called tfdata.